indy_client/test/cli/test_send_node_validation.py

```diff
@@ -1,609 +0,0 @@
-import pytest
-from plenum.common.signer_did import DidSigner
-
-from stp_core.crypto.util import randomSeed
-
-from plenum.common.constants import NODE_IP, NODE_PORT, CLIENT_IP, CLIENT_PORT, \
-    ALIAS, SERVICES, VALIDATOR
-from plenum.common.signer_simple import SimpleSigner
-from plenum.common.util import cryptonymToHex, randomString
-from indy_client.test.cli.conftest import newStewardCli as getNewStewardCli, \
-    newStewardVals as getNewStewardVals, newNodeVals as getNewNodeVals
-from indy_client.test.cli.constants import NODE_REQUEST_COMPLETED, NODE_REQUEST_FAILED, INVALID_SYNTAX
-from indy_client.test.cli.helper import addAgent
-
-NYM_ADDED = "Nym {remote} added"
-
-
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
integrated